api/v1beta1/common_types.go

```diff
@@ -112,120 +112,20 @@ type BarbicanComponentTemplate struct {
 // This SecretStore type is used by the EnabledSecretStores variable inside the specification.
 type SecretStore string
```

Comment:
lets add the following const here and we could replace the hard coded strings in the controllers with those, like `if slices.Contains(instance.Spec.EnabledSecretStores, barbicanv1beta1.SecretStorePKCS11) {`

```
const (
	// SecretStoreSimpleCrypto -
	SecretStoreSimpleCrypto SecretStore = "simple_crypto"

	// SecretStorePKCS11 -
	SecretStorePKCS11 SecretStore = "pkcs11"
)
```

Reply:
Dine